### ISSUE TITLE
????

### ISSUE THREAD

**Issue by Hwzsnow** [xdsarkar/Memristor_Logic#1]
This is orcad capture lib?

**Comment by xdsarkar** [comment]
Yes. I have saved the models and uploaded it. I just have to update the library of the model that I wrote so that others can run my saved models.

**Comment by xdsarkar** [comment]
Do you want it?

**Comment by Hwzsnow** [comment]
So, I don't find it
